components/ImageCropper/src/Dependencies.props

```diff
@@ -21,7 +21,7 @@
 
     <!-- WinUI 3 / WinAppSdk -->
     <ItemGroup Condition="'$(IsWinAppSdk)' == 'true'">
-        <PackageReference Include="Microsoft.Graphics.Win2D" Version="1.0.0.30"/>
+        <PackageReference Include="Microsoft.Graphics.Win2D" Version="1.2.1-experimental2"/>
```

Comment:
Do we know when there won't be an experimental version of this package?

Reply:
@Sergio0694 do you know?

Reply:
Will try to publish it in the next week 🙂